DTNL.LL.Logic/ProjectService.cs

```diff
@@ -71,5 +71,10 @@ public IEnumerable<Project> GetProjectsWithSpecificProjectName(string projectNam
         {
             return GetSpecifiedProjects(p => p.ProjectName.Contains(projectName));
         }
+
+        public async Task<IEnumerable<Project>> GetActiveProjects()
+        {
+            return await _unitOfWork.Projects.GetActiveProjects();
```

Comment:
If you do only one thing, you can beter return the task and make the caller decide how to use the async part then await the one call and give back only the answer to the caller.